Documentation/DataTypes/Properties/GetText.rst.txt

```diff
@@ -634,6 +636,11 @@ siteLanguage
       
       page.10 = TEXT
 	   page.10.dataWrap = The current site language direction is {siteLanguage:direction}
+      
+      # Website title for the current language with fallback to the website title of the site configuration.
+      page.10 = TEXT
+      page.10.data = site:websiteTitle
```

Comment:
Please use the default fallback option instead like shown in Slack.

Reply:
Done 👍